test-network-function/generic/accesscontrol.go

```diff
@@ -54,9 +75,148 @@ var _ = ginkgo.Describe(accessControlTestKey, func() {
 			testRoles(containerUnderTest.oc.GetPodName(), containerUnderTest.oc.GetPodNamespace())
 		}
 
+		// Former "container" tests
+		defer ginkgo.GinkgoRecover()
+		ginkgo.When("a local shell is spawned", func() {
+			goExpectSpawner := interactive.NewGoExpectSpawner()
+			var spawner interactive.Spawner = goExpectSpawner
+			context, err = interactive.SpawnShell(&spawner, defaultTimeout, interactive.Verbose(true))
+			gomega.Expect(err).To(gomega.BeNil())
+			gomega.Expect(context).ToNot(gomega.BeNil())
+			gomega.Expect(context.GetExpecter()).ToNot(gomega.BeNil())
+		})
+		// Query API for certification status of listed containers
+		ginkgo.When("getting certification status", func() {
```

Comment:
does this test that base image is certified ? which container does this code tests ?